docs/management/sentinel.md

```diff
@@ -117,7 +117,7 @@ One set of instances is called `mymaster`, and the other `resque`.
 
 The meaning of the arguments of `sentinel monitor` statements is the following:
 
-    sentinel monitor <master-group-name> <ip> <port> <quorum>
+    sentinel monitor <master_name> <ip> <port> <quorum>
```

Comment:
i see in sentinel.conf, the example is:
```
# sentinel monitor <master-name> <ip> <redis-port> <quorum>
```

so let change it to this?
```suggestion
    sentinel monitor <master-name> <ip> <port> <quorum>
```

also, please scan the sentinel.md file, i see there are some `master-group-name` word, i think we can change it both